.github/workflows/tests.yml

```diff
@@ -240,30 +262,18 @@ jobs:
           cache-dependency-glob: |
             pyproject.toml
 
-      - name: Build package
-        run: |
-          uv pip install build
-          make dist
-
-      - name: Install wheel; test CLI + models with assets
+      - name: Build from source
         run: |
-          uv venv .venv-whl
-          PYTHON_BIN=.venv-whl/bin/python
-          uv pip install --python=$PYTHON_BIN zamba@$(find dist -name 'zamba*.whl') --force-reinstall
-          $PYTHON_BIN -m zamba --help
-          $PYTHON_BIN -c "from zamba.data.video import MegadetectorLiteYoloX; MegadetectorLiteYoloX()"
+          uv pip install .
 
-      - name: Install source; test CLI + models with assets
+      - name: Test import and model loading
         run: |
-          uv venv .venv-sdist
-          PYTHON_BIN=.venv-sdist/bin/python
-          uv pip install --python=$PYTHON_BIN zamba@$(find dist -name 'zamba*.tar.gz') --force-reinstall
-          $PYTHON_BIN -m zamba --help
-          $PYTHON_BIN -c "from zamba.data.video import MegadetectorLiteYoloX; MegadetectorLiteYoloX()"
+          python -m zamba --help
+          python -c "from zamba.data.video import MegadetectorLiteYoloX; MegadetectorLiteYoloX()"
 
   notify:
     name: Notify failed build
-    needs: [code-quality, set-up-os-matrix, tests, test-install]
+    needs: [code-quality, set-up-os-matrix, test-suite, test-install]
```

Comment:
[nit] don't love this naming since in the first "test" is an adjective and in the second it's a verb